tests/aws-cpp-sdk-core-tests/utils/crypto/HashTest.cpp

```diff
@@ -118,3 +119,25 @@ TEST_F(HashUpdateTest, TestCRC32CUpdate)
     Aws::String hexHash = HashingUtils::HexEncode(digest);
     ASSERT_STREQ("fb5b991d", hexHash.c_str());
 }
+
+TEST_F(HashUpdateTest, TestCRC64Update)
+{
+    Crypto::CRC64 hash;
+
+    unsigned char buffer[Crypto::Hash::INTERNAL_HASH_STREAM_BUFFER_SIZE];
```

Comment:
can use 
unsigned char buffer[Crypto::Hash::INTERNAL_HASH_STREAM_BUFFER_SIZE] = {};
to accomplish the same more concisely